ios/Runner/AppDelegate.swift

```diff
@@ -23,12 +26,17 @@ let apnTokenKey = "apnToken"
         let twakeApnChannel = FlutterMethodChannel(
             name: "twake_apn",
             binaryMessenger: controller.binaryMessenger)
-        twakeApnChannel.setMethodCallHandler({
-          (call: FlutterMethodCall, result: @escaping FlutterResult) -> Void in
-            if call.method == "getToken" {
+        twakeApnChannel.setMethodCallHandler { [weak self ] call, result in
+            switch call.method {
+            case "getToken":
                 result(UserDefaults.standard.string(forKey: apnTokenKey))
+            case "getInitialNoti":
+                result(self?.initialNotiInfo)
```

Comment:
```
if let notificationInfo = initialNotiInfo as? [String: Any] {
    // Handle initialNotiInfo not nil
} else {
    // Handle initialNotiInfo is nil
}      
```

IMO: `result(self?.initialNotiInfo)` It can result if `initialNotiInfo ` is nil? Can you explain this case? WDYT my code ?

Reply:
checked in dart code, empty mean no initial noti